micro-osd.sh

```diff
@@ -67,6 +69,16 @@ osd journal size = 100
 osd objectstore = memstore
 osd class load list = *
 osd class default list = *
+
+[client.rgw.${RGW_ID}]
+rgw dns name = test_ceph_a
```

Comment:
```suggestion
rgw dns name = ${HOSTNAME}
```
This is one step closer to getting things to work on both single- and multi-container. It's also just cleaner. I only caught it by trying to run the tests locally.

Reply:
I'm still skipping the single cluster test though since it's going away soon.

Reply:
I'd argue that even with multi-cluster tests as the only test (and it's only going away for the CI) its still not correct as it hard codes the name of only one of the multi cluster hosts, changing it to match the other parameterized values in micro-osd means that it can simply "just work" in more situations in the future. For example, if we were to add rgw mirroring support (regardless of whether that actually exists or not today).